### ISSUE TITLE
How to do this in Pad?

### ISSUE THREAD

**Issue by Ityumenev** [igniterealtime/pade#268]
![scr11](https://github.com/igniterealtime/pade/assets/133771253/1494580b-3cac-4514-9474-b60348fa6005)

**Comment by deleolajide** [comment]
![image](https://github.com/igniterealtime/pade/assets/110731/632ad9d9-2d91-41b0-8890-df35bf603b11)


![image](https://github.com/igniterealtime/pade/assets/110731/91b4a761-a748-414c-b2c8-08a55aceb6d7)

**Comment by Ityumenev** [comment]
Thank you very much. Just today I installed this wonderful plugin and to be honest, I have a lot of questions.
1. How did you put pictures on group chats?
2. I noticed that the contact filter sometimes does not display users, but shows the group - "bots" where one user is located and that's me! I click the "refresh" arrow button and all users show up. Is this some kind of bug?
3. The same thing if you put the status "Online". Sometimes correctly shows sometimes empty. It is solved when you put the filter by name and all at once appear.
4. I created a common room for the company. And I have to give everyone an invitation. In open air, I put a user group on the room that they are members of this room (200 users), but they themselves do not appear in the group in Pade. How do people do it right?
